pkg/apis/pipeline/v1beta1/pipelinerun_conversion_test.go

```diff
@@ -446,23 +450,22 @@ func TestPipelineRunConversionFromDeprecated(t *testing.T) {
 					t.Errorf("ConvertFrom() = %v", err)
 				}
 				t.Logf("ConvertFrom() = %#v", got)
-				if d := cmp.Diff(test.want, got); d != "" {
+				if d := cmp.Diff(test.want, got, cmpopts.EquateEmpty()); d != "" {
 					t.Errorf("roundtrip %s", diff.PrintWantGot(d))
 				}
 			})
 		}
 	}
 }
 
-// Assumptions: The `embedded-status` is defaulted as `Full` as in config.DefaultEmbeddedStatus:
 func TestPipelineRunConversionEmbeddedStatusRoundTrip(t *testing.T) {
 	taskRuns["tr-0"] = trs
 	runs["r-0"] = rrs
 
 	tests := []struct {
-		name string
-		in   *v1beta1.PipelineRun
-		want *v1beta1.PipelineRun
+		name           string
```

Comment:
just wonder if it makes sense to make the `name` of each test case more descriptive? Now they are `taskRuns`, `runs`. etc. We could consider including what `embeddedStatus` is set in each case?

Reply:
done